os/os.module.go

```diff
@@ -0,0 +1,224 @@
+			_k, err := py.ReprAsString(args[0])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			key := strings.ReplaceAll(_k, "'", "") // required
+			_v, err := py.ReprAsString(args[1])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			value := strings.ReplaceAll(_v, "'", "")
+
+			err = os.Setenv(key, value)
+			if err != nil {
+				return nil, py.ExceptionNewf(py.OSError, "Unable to set enviroment variable")
+			}
+			return py.None, nil
+		} else {
+			return nil, py.ExceptionNewf(py.TypeError, "Expected 2 arguments of type string")
+		}
+	} else {
+		return nil, py.ExceptionNewf(py.TypeError, "Expected 2 arguments of type string")
+	}
+}
+
+// Unset (delete) the environment variable named key.
+func unsetenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) == 1 {
+		if reflect.TypeOf(args[0]).String() == "py.String" {
+			_k, err := py.ReprAsString(args[0])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			key := strings.ReplaceAll(_k, "'", "") // required
+			err = os.Unsetenv(key)
+			if err != nil {
+				return nil, py.ExceptionNewf(py.OSError, "Unable to unset enviroment variable")
+			}
+			return py.None, nil
+		} else {
+			return nil, py.ExceptionNewf(py.TypeError, "Expected 1 argument of type string")
+		}
+	} else {
+		return nil, py.ExceptionNewf(py.TypeError, "Expected 1 argument of type string")
+	}
+}
+
+// os._exit() immediate program termination; unline sys.exit(), which raises a SystemExit, this function will termninate the program immediately.
+func _exit(self py.Object, args py.Tuple) (py.Object, error) { // can never return
+	if len(args) == 0 {
+		os.Exit(0)
+		return nil, nil
+	} else if len(args) == 1 {
+		_ec, err := py.GetInt(args[0])
+		if err != nil {
+			os.Exit(1)
+		}
+		exit_code, err := _ec.GoInt()
+		if err != nil {
+			os.Exit(1)
+		}
+		os.Exit(exit_code)
+		return nil, nil
+	} else {
+		os.Exit(1)
+		return nil, nil
+	}
+}
+
+// os.system(command string) this function runs a shell command and directs the output to standard output.
+func system(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) == 0 {
+		return nil, py.ExceptionNewf(py.TypeError, "Expected 1 or more arguments all of type string")
+	} else {
+		var cargs []string
+		if reflect.TypeOf(args[0]).String() == "py.String" {
+			_carg, err := py.ReprAsString(args[0])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			carg := strings.ReplaceAll(_carg, "'", "") // required
+
+			cargs = strings.Split(carg, " ")
+		} else {
+			return nil, py.ExceptionNewf(py.TypeError, "Expected 1 or more arguments all of type string")
+		}
+		command := exec.Command(cargs[0])
```

Comment:
Not sure of why you are passing only `cargs[0]` instead of `exec.Command(cargs[0], cargs[1:]...)`